src/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,376 @@
+import React from 'react'
+import {Box} from '..'
+import {useResponsiveValue, ResponsiveValue} from '../hooks/useResponsiveValue'
+import {SxProp, merge, BetterSystemStyleObject} from '../sx'
+import Heading from '../Heading'
+import {ArrowLeftIcon} from '@primer/octicons-react'
+import Link from '../Link'
+import {ForwardRefComponent as PolymorphicForwardRefComponent} from '../utils/polymorphic'
+
+const REGION_ORDER = {
+  ContextArea: 0,
+  TitleArea: 1,
+  Description: 2,
+  Navigation: 3
+}
+
+// Types that are shared between sub components
+export type sharedPropTypes = {
+  visible?: boolean | ResponsiveValue<boolean>
+} & SxProp
+
+// Default state for the `visible` prop when a sub component is only visible on narrow viewport
+const onlyVisibleOnNarrowView = {
+  narrow: true,
+  regular: false,
+  wide: false
+}
+
+// Default state for the `visible` prop when a sub component is visible on regular and wide viewport
+const visibleOnRegularView = {
+  narrow: false,
+  regular: true,
+  wide: true
+}
+
+// Root
+// -----------------------------------------------------------------------------
+export type PageHeaderProps = {
+  'aria-label'?: React.AriaAttributes['aria-label']
+  as?: React.ElementType
+} & sharedPropTypes
+
+const Root: React.FC<React.PropsWithChildren<PageHeaderProps>> = ({children, sx = {}}) => {
+  const rootStyles = {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '8px',
+    // 24px for wide and regular, 16px for narrow viewports
+    padding: useResponsiveValue(
+      {
+        narrow: 3,
+        regular: 4,
+        wide: 4
+      },
+      false
+    )
+  }
+  return <Box sx={merge<BetterSystemStyleObject>(rootStyles, sx)}>{children}</Box>
+}
+
+// PageHeader.ContextArea : Only visible on narrow viewports by default to provide user context of where they are at their journey. `visible` prop available
+// to manage their custom visibility but consumers should be careful if they choose to hide this on narrow viewports.
+// PageHeader.ContextArea Sub Components: PageHeader.ParentLink, PageHeader.ContextBar, PageHeader.ContextAreaActions
+// ---------------------------------------------------------------------
+
+const ContextArea: React.FC<React.PropsWithChildren<PageHeaderProps>> = ({
+  children,
+  visible = onlyVisibleOnNarrowView,
+  sx = {}
+}) => {
+  const isVisible = useResponsiveValue(visible, false)
+  const contentNavStyles = {
+    display: isVisible ? 'flex' : 'none',
+    flexDirection: 'row',
+    alignItems: 'center',
+    gap: '8px',
```

Comment:
Super random, do we need to use a spacing token like from: https://primer.style/primitives/spacing#size for gaps or are those not available yet?

Separately, should the unit here be `rem` instead of `px` or do we use `px` for layouts?

Reply:
Sorry for not answering to everything you mentioned! I use the `get` function to get the spacing from the "primitives" (from theme-preval to be precise). 

Re `rem` vs `px` I am not sure, it makes sense to use rem but I couldn't find any design guidelines around units. I am a bit confused how I could apply `rem` using the values from "theme-preval" as well.  I'll check this with design first to see how do we want the spaces to be.

Reply:
@mperrotti could I please get your thoughts on that? 🙏🏼

Reply:
Everything in primitives is in rem units now, but we're still testing how this behaves in dotcom– that's why we don't have any official guidance on it yet. Its very safe to use rem units though, so I would go for it! 

We're also not clear yet on the CSS strategy for PRC, so I hesitate to start integrating the new sizing tokens in. We're still making breaking changes to the names and might switch to using CSS variables next quarter in React. I would just use their value in rem units for now until we can roll out primitives to PRC.

Reply:
Oh thank you! That is very helpful.